tests/minimax.wcon

```diff
@@ -0,0 +1,39 @@
+{
+    "tracker-commons":true,
+    "metadata":{
+           "lab":{"location":"CRB, room 5020", "name":"Behavioural Genomics" },
+           "who":"Firstname Lastname",
+           "timestamp":"2012-04-23T18:25:43.511Z",
+           "temperature":{ "experiment":22, "cultivation":20, "units":"C" },
+           "humidity":{ "value":40, "units":"%" },
+           "dish":{ "type":"petri", "size":35, "units":"mm" },
+           "food":"none",
+           "media":"agarose",
+           "sex":"hermaphrodite",
+           "stage":"adult",
+           "age":"18:25:43.511",
+           "strain":"CB4856",
+           "image_orientation":"imaged onto agar",
+           "protocol":"text description of protocol",
+           "software":{
+                "tracker":{ "name":"Software Name", "version":"1.3.0"},
+                "featureID":"@OMG"
+           },
+           "settings":"Any valid JSON entry with hardware and software configuration can go here"
+     },
+    "units":{"t":"s", "x":"mm", "y":"m"},
+    "data":[
+        {"@OMG":{
```

Comment:
The spec does say that "id" is required:

> A data entry is an object with four required keys.

However, later, in the section on custom features, there seem to be three places they can be located.  They must all start with "@".

1. As a top-level object with a key starting with "@"
2. As an element of the "data" array.  This is the one you are concerned about, I believe.
3. As a property of an element of the "data" array, alongside "id", "t", "x", "y".

You can see all three examples in this code excerpt from the spec:

> ```JSON
{
    "tracker-commons":true,
    "units":{"t":"s", "x":"mm", "y":"mm"},
    "data":[
        {"@OMG":{
               "feature_names":["speed", "curvature", "width"],
               "feature_units":["mm/s", "1/mm", "mm"]
        }},
		{ "id":1,
          "t":1.3,
          "x":[1215.11, ...],
          "y":[234.89, ...],
          "@OMG":{ "feature_values":[0.34, 1.5, 0.873] }
        }
        ...
    ],
    "@OMG":{
          "plate_feature_names":["density", "aggregate number"],
          "plate_feature_units":["1/mm^2", null],
          "plate_feature_values":[0.1, 8]
    }
}
```

I would be in favour of simplifying the spec to include just type 1, the top-level custom feature, because in that top-level custom feature other labs could easily construct a structure parallel to the "data" object, giving all the same time-series data, or any other kind of data they want.

Please let me know if you agree and I'll modify the spec document and my code.